src/AddTodoForm.jsx

```diff
@@ -1,31 +1,44 @@
-function AddTodoForm(props) {
+import { useState } from "react";
 
-    function handleAddTodo(event) {
-        event.preventDefault();
+function AddTodoForm({ onAddTodo }) {
+    const [todoTitle, setTodoTitle] = useState('');
 
-        const todoTitle = event.target.title.value;
+    function handleTitleChange(event) {
+        const newTodoTitle = event.target.value; // Retrieve input value from event object
+        
+        // Call the state setter to update the todo title
+        setTodoTitle(newTodoTitle);
+        console.log(newTodoTitle);
```

Comment:
Generally we should just use log lines for debugging.  They don't need to be checked in to the main branch.  As the code base gets big, old log lines can make debugging hard because they clog up your console.  There's also a potential security issue if this were a professional product.

Reply:
ok